gas/config/tc-riscv.c

```diff
@@ -143,6 +143,18 @@ static const struct riscv_ext_version ext_version_table[] =
   {"zba",   ISA_SPEC_CLASS_DRAFT, 0, 93},
   {"zbc",   ISA_SPEC_CLASS_DRAFT, 0, 93},
 
+  {"k",   ISA_SPEC_CLASS_DRAFT, 0, 90},
```

Comment:
Same as above, if k, zkn and zks are expansions, then we don't need to add the default versions for them.

Reply:
Okay, I deleted the version for k,zkn,zks.

Reply:
Sorry that may have misunderstood you.  I meant this is OK for riscv-binutils-experiment, but could be improved in the FSF binutils.  Therefore, we don't need to care about the k and p's expansions in the riscv-binutils-experiment.

Reply:
Ok, let me delete the expansions with k,zkn,zks in the riscv-binutils-experiment..